packages/litexa-html/README.md

```diff
@@ -45,58 +41,147 @@ project_dir
         └── html
 ```
 
-**WARNING**: Unlike other Litexa extensions which conditionally add the
-interface declaration based on whether or not they were used in your skill, the
-`@litexa/html` extension will always add the HTML interface to your skill. Thus, if you
-do not plan on using them for all skills, it may be best to use the local installation
-option for each project.
 
-## Litexa's Alexa Web API Interface
 
-Integrating your skill with the Alexa Web API will allow it to both send and receive data
-from your web app.
+### `HTML` statement
+
+This injects an `Alexa.Presentation.HTML.Start` directive into the current response, 
+which causes a device to load the given url. This statement will do nothing in the
+absence of the HTML interface, so it's safe to let it silently fail.
+
+The presence of this statement will also automatically add the `ALEXA_PRESENTATION_HTML`
+to your skill manifest.
+
+```coffeescript
+launch
+  HTML "app.html"
+    timeout: 200
```

Comment:
I'm going to defer to the directive documentation on that, just in case the specifics of the meaning evolve over time. I don't really intend for this statement to shadow knowledge of the directive, I feel like that's an abstraction that would leak relatively quickly.